.github/workflows/linux-aarch64.yml

```diff
@@ -34,7 +34,7 @@ jobs:
       runner_prefix: ${{ needs.get-label-type.outputs.label-type }}
       build-environment: linux-jammy-aarch64-py3.10
       docker-image-name: pytorch-linux-jammy-aarch64-py3.10-gcc11
-      runner: linux.arm64.2xlarge
+      runner: linux.arm64.m7g.4xlarge
```

Comment:
It's fine to use more powerful HW to speedup the build, but it should not affect which features PyTorch binary can use.